pyproject.toml

```diff
@@ -29,62 +29,62 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4"
-asyncio-dgram = "2.1.2"
-dnspython = "2.3.0"
+python = ">=3.8.1,<4"
+asyncio-dgram = "^2.1.2"
+dnspython = "^2.3.0"
 
 [tool.poetry.group.dev.dependencies]
-pre-commit = "^3.2.2"
-tox = "^3.26.0"
-tox-poetry = ">=0.4.1,<0.6.0"
+pre-commit = "^3.3.2"
+tox = "^3.28.0"
+tox-poetry = "0.5.0" # deprecated, no sense in `^`
```

Comment:
Also, I have staged local changes that remove tox entirely once the automated releases are happening in GitHub actions. So, we do not need to concern ourselves with tox-related things.

Reply:
It means that I have incomplete local progress for #488